Source/UI/Controllers/ThreedsWebViewController.swift

```diff
@@ -53,42 +52,67 @@ public class ThreedsWebViewController: UIViewController,
     public override func viewDidLoad() {
         super.viewDidLoad()
 
-        guard let authUrl = url else { return }
-        let myURL = URL(string: authUrl)
-        let myRequest = URLRequest(url: myURL!)
+        guard let authUrl = url,
+              let url = URL(string: authUrl) else {
+            return
+        }
+
+        let myRequest = URLRequest(url: url)
         webView.navigationDelegate = self
         webView.load(myRequest)
     }
+}
 
-    // MARK: - WKNavigationDelegate
+// MARK: - WKNavigationDelegate
+extension ThreedsWebViewController: WKNavigationDelegate {
 
     /// Called when the web view begins to receive web content.
     public func webView(_ webView: WKWebView, didCommit navigation: WKNavigation!) {
-        shouldDismiss(absoluteUrl: webView.url!)
+
+        guard let absoluteUrl = webView.url else {
+            return
```

Comment:
Is it safe to do silent returns here ?

Reply:
I think that a silent return is better than an app crash, which was what we had before. We're also limited here on return type as this is a WKNavigationDelegate protocol function.